www/gatsby-node.js

```diff
@@ -18,3 +18,16 @@ exports.createPages = ({ graphql, actions, reporter }) => createPages(graphql, a
 exports.sourceNodes = ({ actions, createNodeId, createContentDigest }) => {
   createCssUtilityClassNodes({ actions, createNodeId, createContentDigest });
 };
+
+exports.onCreatePage = ({ page, actions }) => {
```

Comment:
Since we have a util function in a separate file for every gatsby-node hook, maybe create `./utils/onCreatePage` file and put this function there to be consistent?

Also, I think since you add `githubEditPath` here you can remove similar code from `createPages` function that we added for component pages and just use this solution for all pages.

Reply:
Yes, thanks 👍